src/cellfunction/eval_point.jl

```diff
@@ -216,3 +216,23 @@ function change_domain(p::FacePoint{ReferenceDomain}, ::PhysicalDomain)
     x_phy = _apply_mapping(m, get_coords(p))
     FacePoint(x_phy, faceInfo, PhysicalDomain())
 end
+
+ElementPoint(x, elementInfo::CellInfo, ds) = CellPoint(x, elementInfo, ds)
```

Comment:
Since you've started this, can you go one step further and merge https://github.com/bcube-project/Bcube.jl/blob/67f5fe6db20650887e6c0e858e86e51174f3cc43/src/integration/integration.jl#L22 and https://github.com/bcube-project/Bcube.jl/blob/67f5fe6db20650887e6c0e858e86e51174f3cc43/src/integration/integration.jl#L32 ?

Reply:
Actually I have already done it in the next PR